#!/usr/bin/python3

def raise_exception_msg(message=""):
    """raise a NameError exception with message."""
    raise NameError(message)
